src/SASjsApiClient.ts

```diff
@@ -118,18 +131,30 @@ export class SASjsApiClient {
     code: string,
     runTime: string = 'sas',
     authConfig?: AuthConfig
-  ) {
+  ): Promise<ScriptExecutionResult> {
     const access_token = await this.getAccessTokenForRequest(authConfig)
 
-    let parsedSasjsServerLog = ''
+    let executionResult: ScriptExecutionResult = { log: '' }
 
     await this.requestClient
       .post('SASjsApi/code/execute', { code, runTime }, access_token)
       .then((res: any) => {
-        if (res.log) parsedSasjsServerLog = res.log
+        const { log, printOutput } = res
```

Comment:
this can be written as 
```suggestion
        const { log, printOutput, result: webout } = res
```